src/jarabe/screenshotpanel/gui.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python
```

Comment:
Can you move this to jarabe.gui.screenshotpanel or something?  Having a whole folder for 1 file is unnecessary.